app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,198 @@
+<resources>
+    <string name="app_name">Jerboa f[r Lemmy</string>
```

Comment:
I think this should be "für" instead of "f[r"

Reply:
You are totally right. Fixed it just now.